.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Build
+on:
+  push:
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: 'ubuntu-latest' 
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.x.x'
+          cache: 'npm'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install deps
+        run: |
+          npm ci
+
+      - name: Build and test 
+        run: |
+          npm run dist
+
+      - name: Publish 
+        if: ${{github.ref == 'refs/heads/master' || startsWith(github.ref, 'ref/heads/hotfix-')}}
```

Comment:
this looks wrong...

first, it says `ref/` instead of `refs/`, but any branch creation with no approval would be published?

Reply:
There is a BPR for the `hotfix-` pattern